src/SimpleMdeReact.tsx

```diff
@@ -67,46 +62,7 @@ export type IndexEventsSignature = {
 export interface SimpleMdeToCodemirrorEvents
   extends CopyEvents,
     GlobalEvents,
-    IndexEventsSignature {
-  change?: (instance: Editor, changeObj: EditorChangeLinkedList) => void;
```

Comment:
Replaced redefining all these events w/ just relying on the `EditorEventMap` interface that the new codemirror types export.